package.json

```diff
@@ -48,14 +48,14 @@
     "eject": "react-app-rewired eject"
   },
   "devDependencies": {
-    "enzyme": "^3.6.0",
-    "enzyme-adapter-react-16": "^1.5.0",
+    "enzyme": "^3.7.0",
+    "enzyme-adapter-react-16": "^1.6.0",
     "eslint-plugin-lodash-fp": "^2.1.3",
     "jest": "^23.6.0",
     "mutationobserver-shim": "^0.3.2",
     "raw-loader": "^0.5.1",
     "react-app-rewire-eslint": "^0.2.3",
-    "react-app-rewire-hot-loader": "^1.0.1",
+    "react-app-rewire-hot-loader": "1.0.1",
```

Comment:
Pinning this explicitly because newer versions don't include webpack, and so they generate an error that I'll have to figure out before upgrading.